std_msgs/Float32MultiArray state
std_msgs/Float32MultiArray target
std_msgs/Float32MultiArray constraints
float32 T_horizon
--- 
bool                       success
string                     error
std_msgs/Float32MultiArray opt_state
std_msgs/Float32MultiArray opt_input
std_msgs/Float32MultiArray opt_feedback
